lib/rules/await-fire-event.ts

```diff
@@ -29,7 +29,7 @@ export default createTestingLibraryRule<Options, MessageIds>({
     },
     messages: {
       awaitFireEvent:
-        'Promise returned from `fireEvent.{{ methodName }}` must be handled',
+        'Promise returned from `fireEvent.{{ name }}` must be handled',
       fireEventWrapper:
         'Promise returned from `fireEvent.{{ wrapperName }}` wrapper over fire event method must be handled',
```

Comment:
I think we have to do the same change here, right?

Reply:
I think the same, but change wrapperName to name breaks the tests, i will try to find the problem and fix that :D

Reply:
Let me know if you need help for this!

Reply:
You are right! It should be the name of the wrapper 👌